JWT.sln

```diff
@@ -38,6 +38,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "JWT.Tests.Core2", "tests\JW
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "JWT.Tests.Core3", "tests\JWT.Tests.Core3\JWT.Tests.Core3.csproj", "{559F1CBB-DB9C-4149-BEAC-417DA1BBB162}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "JWT.Tests.Net35", "tests\JWT.Tests.Net35\JWT.Tests.Net35.csproj", "{30D3D291-2C79-4CDD-94D3-C834A4995149}"

```

Comment:
Please rename old .NET Framework tests project to `Net472` then. OCD :) aka Consistency Is King.

Reply:
And move both projects together: common first, then .net, then .net core, please.